frontend/types/src/components/Examination/ExaminationForm.vue.d.ts

```
@@ -0,0 +1,141 @@
+import type { Examination, SubcategoryMap } from '@/stores/examinationStore';
+declare const _default: import("vue").DefineComponent<{}, {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)